modules/concurrency/src/spinner.cpp

```diff
@@ -89,4 +89,8 @@ void Spinner::addStopCallback(std::function<void()>&& callback) {
   stop_callback_ = std::move(callback);
 }
 
+auto Spinner::spinCount() const -> uint64_t {
+  return spin_count_;
```

Comment:
its never incremented and the user has no idea that he needs to in `spinOnce`... Should we wrap spinOnce into a `spinOnceParent` (very bad name) sth like
```
void spinOnceParent() {
   ++spin_count_;
   spinOnce();
}
```

Reply:
at the very least, we need to leave a comment

Reply:
it is incremented in `spin()` line 64

Reply:
You actually increment it in `spin()`
`    const auto target_timestamp = start_timestamp_ + (++spin_count_) * spin_period_;`

Reply:
sorry, missed that